src/Catty/DataModel/Bricks/Control/RepeatUntilBrick.m

```diff
@@ -69,13 +69,6 @@ - (NSString*)description
     return [NSString stringWithFormat:@"RepeatLoop"];
 }
 
-#pragma mark - Copy
-- (id)mutableCopyWithContext:(CBMutableCopyContext*)context
```

Comment:
The _RepeatUntilBrick_ contains only one Formula. When copying a Brick, there should also be a copy of the Formula, so that the Formula can be edited separately (in the original and the copied Brick). This happens all in the parent method, so no overriding necessary. Thanks!